rock_paper_scissors.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rock Paper Scissors</title>
+</head>
+<body>
+    <table border="1">
+        <tr>
+            <th colspan="3">You Vs Computer</th>
+        </tr>
+        <tr>
+            <th>Computer:</th>
+            <td colspan="2"><span id="computer"></span></td>
+        </tr>
+        <tr>
+            <th >Winner:</th>
+            <td colspan="2"><span id="winner"></span></td>
+        </tr>
+        <tr>
+            <th>Score</th>
+            <td colspan="2"><span id="user_score">0</span>:<span id="computer_score">0</span></td>
+        </tr>
+        <tr>
+            <td><button id="btn_rock">Rock</button></td>
+            <td><button id="btn_paper">Paper</button></td>
+            <td><button id="btn_scissor">Scissor</button></td>
+        </tr>
+    </table>
+   
+</body>
+<script src="basics.js"></script>
+<script>
+    let game_options=['rock','paper','scissor']//Game options available
+    var btn=[];//created a empty button list
+    for (let i in game_options){
+        /*buttons are given ids i.e. btn_rock,btn_paper,btn_scissor which can be fetched dynamically by adding prefix "btn_" to game_options*/
+        btn.push(document.getElementById("btn_"+game_options[i]))
+
+    }
+    /*the gameoption on left loses and the right one wins*/
+    winning_arrangements=[['rock','paper'],['paper','scissor'],['scissor','rock']]
+    let user_score_ele=document.getElementById("user_score")
+    let computer_score_ele=document.getElementById("computer_score")
+    let user_score=parseInt(user_score_ele.innerHTML)
+    let computer_score=parseInt(computer_score_ele.innerHTML)
+    let computer=document.getElementById("computer")
+    let computerResponse=()=>Math.floor(Math.random()*3)
+    let scoreIncrement=(player)=>{
```

Comment:
make sure to add adequate spacings while using arrow function and assingmnets
e.g 
scoreIncrement = (player) => {
...
}
update the same for all the cases.

Reply:
The spacing issues have been fixed